### PR TITLE
added css to navbar, logo to top of page with title and company logo …

### PR DESCRIPTION
**Pull request by Miska-Mynttinen**
Added css to navigation and crated a navbar, logo to top of page with title and company logo test component on the bottom of the home page (also pictures folder with as logo and test pictures).

Couldn't get position: sticky  ,  working with navbar.

Somehov when company logos are at the bottom of the page and vertical scroll appears, horizontal scroll appears and couldn't remove horizontal scroll.

### CODE REVIEW

**Comment by Miska-Mynttinen** [comment]
Aa nii myös toi yläteksti(title AS 25) meni vähä oudosti ku se on samas komponentis ku se logo vasemmas yläkulmas. File Navigation.

File: client/src/components/nav/Navigation.tsx [hunk 1]
```diff
@@ -1,19 +1,31 @@
...
+            <Logo>
+              <img src={ASLogo} alt="Logo" style={{ width: "54.2px", height: "54.2px" }} />
+            </Logo>
+            <TitleText>
+              AS-Kilta 25 vuosijuhla
+            </TitleText>
+        </LogoBanner>
       {/* A "layout route" is a good place to put markup you want to
             share across all the pages on your site, like navigation. */}
       <nav>
```
Comment (coodeex) [inline client/src/components/nav/Navigation.tsx:18]: you can remove <nav>

File: client/src/components/nav/Navigation.tsx [hunk 2]
```diff
@@ -35,3 +47,44 @@ export const Navigation = () => {
...
+`;
+
+const TitleText = styled.div`
+  display: flex;
+  margin: auto;
+  align-self: center;
+  font-size: 22px;
+`;
+
+const NavBar = styled.div`
```
Comment (coodeex) [inline client/src/components/nav/Navigation.tsx:69]: and you can change this to styled.nav

and add
position: sticky;
top: 0;

to have a sticky navbar

File: client/src/pages/homeComponents/CompanyLogos.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      <img src={testi2} alt="testi2" style={{ width: "200px", height: "100px" }} />
+      <img src={testi3} alt="testi3" style={{ width: "200px", height: "100px" }} />
+    </LogoList>
+  );
+};
+
+const LogoList = styled.div`
+  padding-top: 500px;
+  display: flex;
+  justify-content: space-evenly;
```
Comment (coodeex) [inline client/src/pages/homeComponents/CompanyLogos.tsx:19]: you can add flex-wrap: wrap; so images don't cause horizontal overflow

**Review by coodeex — APPROVED** [review]
looks good! :+1: feel free to add little fixes mentioned above and merge

**Pull request merged**